app/views/admin/users/_form.html.erb

```diff
@@ -9,21 +9,35 @@
   <div class="row">
     <div class="col-md-6">
       <h3>Partners</h3>
-      <p>Partners this user can edit, as well as their associatied calendars</p>
+      <p>Partners this user can edit, as well as their associated calendars</p>
       <%= f.association :partners,
                         label: false,
                         collection: options_for_partners,
                         selected: @partners,
                         input_html: {class: 'form-control select2' }
           %>
     </div>
+
     <div class="col-md-6">
       <h3>Neighbourhoods</h3>
-      <p>Grants access to every partner and calendar in these neighbourhoods. Use with care!</p>
-      <%= f.association :neighbourhoods,
-                        label: false,
-                        collection: options_for_neighbourhoods,
-                        input_html: {class: 'form-control select2' } %>
+      <p>This user has <%= pluralize @user.neighbourhoods.count, 'neighbourhood' %>.</p>
```

Comment:
I think just change this for a list of them. Maybe this should be the else block for the following? So if root show the picker, if not show the list of neighbourhoods.